app/app.js

```diff
@@ -5,11 +5,15 @@ import Relay from 'react-relay';
 import {
   Router,
   Reducer,
-  Scene
+  Scene,
+  Actions
 } from 'react-native-router-flux';
 import RelayRenderer from './shared/relayComponentRenderer';
 import MyLibrary from './components/myLibrary/myLibrary';
 
+import imgArrowDown from './resources/arrow_down.png';
```

Comment:
as far as I know, this is ES6. I understand this is inconsistent code style since `require` has been used. However, when you look at how this is using: https://github.com/recbook/recbook-client/pull/21/files#diff-2de9eda1655d499e934e7465e87ca0bdR120, I thought doing `(condition) ? img1 : img2` becomes much cleaner code than writing `(condition) ? require('img1 path') : require('img2 path')
feel free to tell me if you have better ideas